ecommerce/tasks.py

```diff
@@ -0,0 +1,54 @@
+"""
+Ecommerce tasks
+"""
+from django.core.exceptions import ObjectDoesNotExist
+
+from ecommerce.hubspot_api import (
+    send_hubspot_request,
+    make_contact_sync_message,
+    make_product_sync_message,
+    make_deal_sync_message,
+    make_line_item_sync_message,
+)
+from mitxpro.celery import app
+
+
+HUBSPOT_SYNC_URL = "/extensions/ecomm/v1/sync-messages"
+
+
+@app.task
+def sync_contact_with_hubspot(user_id):
+    """Send a sync-message to sync a user with a hubspot contact"""
+    from users.models import User
+
+    try:
+        user = User.objects.get(id=user_id)
+    except ObjectDoesNotExist:
+        return
+    body = [make_contact_sync_message(user)]
```

Comment:
`make_contact_sync_message` expects `user_id`, not `user`

I changed it in my local instance, and when I run this task I get a 204 status back, but the user does not appear in the list of contacts on hubspot.

Reply:
Still not seeing any new contacts since May 16 at the following url:
https://app.hubspot.com/contacts/.../contacts/list/view/all/

I am submitting users with emails that aren't there yet

Reply:
Full URL is https://app.hubspot.com/contacts/<portal_id>/contacts/list/view/all/?

I ran the management command.  I can see your email/contact generated from May 16 and the contacts I added from the previous PR last week, but no emails for users I've tried to add with this PR.  Maybe I'm just looking in the wrong place?